### PR TITLE
Fix grammar in dealing-with-streams doc

### PR DESCRIPTION
**Pull request by dschaller**

### CODE REVIEW

File: docs/writing-a-plugin/dealing-with-streams.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > It is highly recommended to write plugins supporting streams. Here is some information on creating a gulp plugin that supports streams.
 
-> Make sure to follow the best practice regarding error handling and add the line that make the gulp plugin re-emit the first error caught during the transformation of the content
+> Make sure to follow the best practices regarding error handling and add a line that smake the gulp plugin re-emit the first error caught during the transformation of the content.
```
Comment (phated) [inline docs/writing-a-plugin/dealing-with-streams.md]: smake

**Comment by contra** [comment]
Thanks! My favorite kind of PR to merge.

**Comment by phated** [comment]
Switched the settings to squash merge and going to rebase the changes together.  Cleaner git history to generate changelogs based on.

**Comment by contra** [comment]
@phated ah, forgot about that

**Pull request merged**